pybamm/parameters/bpx.py

```diff
@@ -240,7 +240,7 @@ def _entropic_change(sto, c_s_max, dUdT, constant=False):
     # (cs-cs_max)) in BPX exchange current is defined j0 = F * k_norm * sqrt((ce/ce0) *
     # (cs/cs_max) * (1-cs/cs_max))
     c_e = pybamm_dict["Initial concentration in electrolyte [mol.m-3]"]
-    F = 96485
+    F = 96485.3321233100184
```

Comment:
If the BPX spec does not specify a value, then this is better
```suggestion
    F = pybamm.constants.F.value
```

Note: I did not test this myself